mypy/stubgen.py

```diff
@@ -1271,8 +1271,16 @@ def get_qualified_name(o: Expression) -> str:
 
 
 def remove_blacklisted_modules(modules: List[StubSource]) -> List[StubSource]:
-    return [module for module in modules
-            if module.path is None or not is_blacklisted_path(module.path)]
+    module = []
+    for m in modules:
+        if m.path is not None:
+            if is_blacklisted_path(m.path):
+                print(f"Ignoring Module Path '{m.path}'")
```

Comment:
You don't need to capitalize every word and when people see the path they can probably tell it's a path so you can leave that last part out :wink: 

```suggestion
                print(f"Ignoring module '{m.path}'")
```